common/pom.xml

```diff
@@ -96,6 +96,14 @@
             <groupId>org.jboss.spec.javax.annotation</groupId>
             <artifactId>jboss-annotations-api_1.2_spec</artifactId>
         </dependency>
+
+        <dependency>
+          <groupId>javax.json</groupId>
+          <artifactId>javax.json-api</artifactId>
+          <version>1.0</version>
+          <scope>test</scope>
+        </dependency>
```

Comment:
I tried moving it up and nothing appears to work unless the dependencies are placed in the area I have them now.